pkg/virt-controller/watch/vmi/vmi.go

```diff
@@ -171,6 +172,10 @@ func (i informalSyncError) RequiresRequeue() bool {
 	return false
 }
 
+type statusUpdater interface {
+	Update(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod) error
```

Comment:
WDYT about a different name for the API? Update() is going to make it a little painful to search for k8s apiserver update interactions

Reply:
I'm open for suggestions.
I think that on modern IDEs that won't be a problem, since they are aware of types.

Reply:
This interface was removed.